src/components/artwork_grid.tsx

```diff
@@ -3,14 +3,29 @@ import * as Relay from "react-relay"
 import styled from "styled-components"
 import Artwork from "./artwork/index"
 
-export interface GridProps extends RelayProps, React.HTMLProps<ArtworkGrid> {
+interface Props extends RelayProps, React.HTMLProps<ArtworkGrid> {
   columnCount?: number,
   sectionMargin?: number,
   itemMargin?: number,
+  onLoadMore?: any,
 }
 
-export class ArtworkGrid extends React.Component<GridProps, null> {
-  public static defaultProps: Partial<GridProps>
+interface State {
+  loading: boolean,
+}
+
+export class ArtworkGrid extends React.Component<Props, State> {
+  public static defaultProps: Partial<Props>
+
+  componentDidMount() {
+    window.onscroll = () => {
```

Comment:
I would suggest using `window.addEventListener('scroll')` instead of `window.onscroll` for the following reasons:

- You can have multiple instances of ArtworkGrid on one page and they won't override each others event handlers.
- You can call `window.removeEventListener` in componentDidUnmount to remove the listener